internal/cli/cmd/status.go

```diff
@@ -47,6 +48,10 @@ func newCmdStatus() *cobra.Command {
 			} else {
 				fmt.Print(s.Format())
 			}
+
+			if err == nil && len(s.CollectionErrors) > 0 {
```

Comment:
I think we could do something like:
```go
  return errors.Join(err, errors.Join(s.CollectionErrors...))
```

Reply:
@derailed Thanks for the idea!

Unfortunately this would interfere a little bit with the [overall output](https://github.com/ttretau/cilium-cli/blob/89f65e3e1019be58f5bf00385745045bc50df955/cmd/cilium/main.go#L23) of errors. For example in Status [errors are already in output](https://github.com/ttretau/cilium-cli/blob/89f65e3e1019be58f5bf00385745045bc50df955/status/status.go#L371). Therefore the simple error message.

example output with the join:
```
Containers:            cilium
                       cilium-operator
Cluster Pods:          0/41 managed by Cilium
Helm chart version:
Errors:                cilium    cilium    daemonsets.apps "cilium" not found
daemonsets.apps "cilium" not found